modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee

```diff
@@ -951,21 +950,24 @@ class FileRowSummaryController extends Backbone.View
 			mm = '0' + mm
 		today = '_' + mm + '_' + dd + '_' + yyyy
 
-		# Rename the SDF file representing the current
-		currentFileName = originalFileName.replace "\.sdf", today + "_current_state.sdf"
-    
+		#In some cases, there are artifact .sdf extensions in a file name that cause downloading issues when renanubg a file to .zip
+		#we need to replace ONLY the last .sdf in the file with ".zip" or "{current_date}_current_state.sdf", so we can't use replace which will replace the first
+		fileNameLength = fileName.length #we have to assign this first since for some reason simply using fileName.length - 4 just wont work in line below? 
+		fileNameBase = fileName.slice 0, (fileNameLength)-4 
+	
+		# Rename the SDF file representing the current data
+		currentFileName = fileNameBase + today + "_current_state.sdf"
+
 		# Replace .sdf with .zip since the report file shares the same name 
-		reportName = fileName.replace "\.sdf", ".zip"
-		reportDisplayName = originalFileName.replace "\.sdf", ".zip"
+		reportName = fileNameBase + ".zip"
 
 		toDisplay =
 			fileName: fileName
-			originalFileName: originalFileName
```

Comment:
@hhan-schrodinger I'd added this "originalFileName" stuff as part of ACAS-271. Please don't remove it! If this wasn't working on your local environment, you may need to update Roo, since I merged in the changes for ACAS-271 recently.

Reply:
Ooops, sorry Brian! I assumed it was an artifact/error on my part when I made it.